gprofiler/utils/perf.py

```diff
@@ -3,14 +3,71 @@
 # Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
 #
 
-from gprofiler.exceptions import CalledProcessError
+from enum import Enum
+from pathlib import Path
+from threading import Event
+
+from gprofiler.exceptions import CalledProcessError, PerfNoSupportedEvent
 from gprofiler.log import get_logger_adapter
 from gprofiler.utils import run_process
-from gprofiler.utils.perf_process import perf_path
+from gprofiler.utils.perf_process import PerfProcess, perf_path
 
 logger = get_logger_adapter(__name__)
 
 
+class SUPPORTED_PERF_EVENTS(Enum):
+    PERF_DEFAULT = "default"
```

Comment:
Order is important here, because you want to test `default` first, right? Please comment it then (so no-one changes the order)

Reply:
please review
68b59a2cb3458d430c8fe6a1953c0979ed732557